_posts/2020-03-19-quarkus-1-3-0-final-released.adoc

```diff
@@ -0,0 +1,206 @@
+---
+layout: post
+title: 'Quarkus 1.3.0.Final released - New class loader infrastructure, GraalVM 20 support, and a lot more'
+date: 2020-03-19
+tags: release
+synopsis: Quarkus 1.3.0.Final comes with a new class loader infrastructure, GraalVM 20 support, 8 new extensions and a lot of bugfixes and improvements.
+author: gsmet
+---
+
+It has been a bit less than two months since our last major Quarkus release
+as we went through some extensive testing phases to validate our new class loader infrastructure.
+
+A big thanks to all the people who tested our Alphas/CRs and provided feedback.
+
+Two months?
+That's a lot in the Quarkus world so expect a whole lot of new things.
+
+Enough introduction, let's see what 1.3 brings to the world.
+
+== Migration Guide
+
+Eager to migrate your existing applications? Here is https://github.com/quarkusio/quarkus/wiki/Migration-Guide-1.3[our migration guide for 1.3.0.Final].
+
+== What's new?
```

Comment:
I think a mention to the Kubernetes extension overhaul (with the associated container-image extensions) makes sense here, because it vastly improves the experience when working with Kubernetes.
If you agree and would like me to help out with that, I'd be happy to.

Reply:
I put in under new extensions but feel free to move it elsewhere and write something specific.

I will push the adjustments and then feel free to add a commit to this PR.

Reply:
OK sure. I think we need to call out the Kubernetes extension updates, because the container image extensions on their own don't imply one can deploy easily now to Kubernetes

Reply:
Done, PR is yours!

Reply:
Cool, thanks

Reply:
PR updated